procedures/katello/branch.md.erb

```diff
@@ -43,18 +58,22 @@
   - [ ] [katello](https://github.com/katello/katello)
 - [ ] Bump versions to <%= develop %>-master
   - [ ] [katello](https://github.com/katello/katello)
-- [ ] Create PR to foreman-infra to add a mapping between the current Foreman and Katello release branches. Also remove the oldest mapping from the config. See [this example](https://github.com/theforeman/foreman-infra/pull/1491).
+- [ ] Create PR to [jenkins-jobs](https://github.com/theforeman/jenkins-jobs) to add a mapping between the current Foreman and Katello release branches. Also remove the oldest mapping from the config. See [this example](https://github.com/theforeman/jenkins-jobs/pull/54).
+- [ ] Review the Foreman schedule and planning ([example](https://community.theforeman.org/t/foreman-2-5-schedule-and-planning/22219)) and note the date of the first scheduled release candidate.
+- [ ] Generate and post the release procedure, if not already posted:
+  - [ ] (Note that Katello uses dots instead of dashes for release candidates, e.g. 4.1.1.rc1 not 4.1.1-rc1)
+  - [ ] Make sure to run `./tools.rb setup-environment` first
+  - [ ] Run `./tools.rb procedure release katello <%= release %>.rc1 <rc_release_date>` in [tool_belt](https://github.com/theforeman/tool_belt)
+  - [ ] Post the output in [Development](https://community.theforeman.org/c/development/9) with a "Releases" tag
 - [ ] Branch docs
```

Comment:
This now means branching https://github.com/theforeman/foreman-documentation, right?

Reply:
Yeah, I imagine the Foreman release owner should be doing the steps [here](https://github.com/theforeman/foreman-documentation#branching-new-release), and these steps follow after that.